packages/cli/src/commands/pull.ts

```diff
@@ -35,6 +35,7 @@ const help = () => {
     'DIR'
   )}    Path to the global ${'`.vercel`'} directory
     -d, --debug                    Debug mode [off]
+    --git-branch                   Specify the Git branch to pull specific Environment Variables for
```

Comment:
This seems unrelated to the PR?

Reply:
Yes, unrelated. I found that option was missing when I documented `--git-branch` last week. Want me to move this to a separate PR?